cli/internal/runsummary/run_summary.go

```diff
@@ -41,6 +43,7 @@ const (
 // about the Run and references to other things that we need.
 type Meta struct {
 	RunSummary         *RunSummary
+	ctx                context.Context
```

Comment:
Go idiom: don't save `Context` on a struct, pass it in where you need it. In this case it should be an argument to `Close`. `Context` instances are more properly tied to an operation than a particular data structure.